build.sbt

```diff
@@ -73,14 +92,14 @@ val circeDeps  = Seq(
 )
 
 val otherDeps = Seq(
-  "com.beachape"        %% "enumeratum"  % V.enumeratum,
-  "org.apache.avro"      % "avro"        % V.avro,
-  "com.typesafe"         % "config"      % V.typesafeConfig,
-  "com.google.guava"     % "guava"       % V.guava,
-  "org.typelevel"       %% "squants"     % V.squants,
-  "com.sksamuel.avro4s" %% "avro4s-core" % V.avro4s,
+  "com.beachape"        %% "enumeratum"  % V.enumeratum % Provided,
```

Comment:
Why are all of these being marked as `Provided`? Most of these are required for the code to work, so they need to be there, which means to use this people have to include them explicitly in their own code, which seems like a burden.

Reply:
I made these changes months ago when we were getting started. I can’t remember exactly why, but without using provided on these, the Delivery Streams + flinkrunner.jar would fail to run.

Reply:
Removed it, was not needed.